nifi-extension-bundles/nifi-cdc/nifi-cdc-api/src/main/java/org/apache/nifi/cdc/event/ColumnDefinition.java

```diff
@@ -24,14 +24,16 @@
 public class ColumnDefinition {
 
     private int type;
+    private Boolean isSigned;
```

Comment:
Using the `Boolean` box type is usually unnecessary, and prefixing the property with `is` also brings unnecessary wording. Naming the property `signed` is one option, with a primitive `boolean` type. If the signed status can be undefined, then an `enum` should be used with `SIGNED`, `UNSIGNED`, and `UNDEFINED`.